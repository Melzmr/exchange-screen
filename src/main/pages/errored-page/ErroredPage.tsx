import React from 'react';

export const ErroredPage: React.FC = ({children}) => (
    <div className="page_container">
      {children}
    </div>
);

ErroredPage.displayName = 'ErroredPage';
