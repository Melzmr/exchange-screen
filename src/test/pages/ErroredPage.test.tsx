import {render} from '@testing-library/react';
import {shallow} from 'enzyme';
import {ErroredPage} from '../../main/pages/errored-page/ErroredPage';
import React from 'react';

describe('ErroredPage', () => {
  let comp: React.ReactElement;

  beforeEach(() => {
    comp = (
        <ErroredPage>
          {'Errored'}
        </ErroredPage>
    );
  });

  it('renders ErroredPage component', () => {
    const wrapper = shallow(comp);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders children', () => {
    const {getByText} = render(comp);
    const el = getByText('Errored');
    expect(el).toBeInTheDocument();
  });

});


