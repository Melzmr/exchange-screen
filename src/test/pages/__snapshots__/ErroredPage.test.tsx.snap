// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErroredPage renders ErroredPage component 1`] = `
<div
  className="page_container"
>
  <div
    className="page_content"
  >
    Errored
  </div>
</div>
`;
